guide/payments/receive.md

```diff
@@ -0,0 +1,213 @@
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/address-switch.png"
+   retina = "/assets/images/guide/payments/receive/address-switch@2x.png"
+   alt-text = "Screen with modal of invoice details, and address switcher"
+   caption = "Allow the receiver to switch to a legacy compatible address"
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/address-info.png"
+   retina = "/assets/images/guide/payments/receive/address-info@2x.png"
+   alt-text = "Screen explaining SegWit and Legacy addresses"
+   caption = "Provide information about the different address types"
+   width = 250
+   height = 541
+%}
+
+</div>
+
+<!--
+On /guide/payments/send/#inputting-an-address
+Add below as Do's & Don'ts
+
+> Besides pushing wallets to adopt Bech32, wallets should provide better and clearer error messages to the end-user. Merchants could use P2SH to mitigate. In my opinion P2SH is just a patch not a solution. A solution where Bech32 invoice can fallback to P2SH would be a good balance.
+> @pavelenex
+-->
+
+## Sharing the payment details
+Whether it’s just an address or a payment link, the two methods of sharing the payment details are text or scannable QR Codes.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/share.png"
+   retina = "/assets/images/guide/payments/receive/share@2x.png"
+   alt-text = "Screen showing the invoice with a share button"
+   caption = ""
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/receive/share-sharesheet.png"
+   retina = "/assets/images/guide/payments/receive/share-sharesheet@2x.png"
+   alt-text = "Screen showing the invoice overlayed with a share sheet"
+   caption = ""
+   width = 250
+   height = 541
+%}
+
+</div>
+
+
+**Text**
+
+Sharing payment details as text is as straightforward as copying and pasting. It is also typical to provide the receiver with a button to conveniently share in other applications, with “share sheets,” which most mobile operating systems offer.
+
+**QR Codes**
+
+QR Codes are also a common way to exchange payment details when the two devices are near one another. It is not without its faults though, as some people may not be familiar with QR Codes, and lower-end phones may have difficulty reading them.
+
+Here are some constraints to consider when displaying QR Codes:
+
+- QR Codes with more blocks (modules) are challenging to read for devices with low-resolution cameras.
+- There should be sufficient contrast between the blocks in a QR Code and the background. Ideally, keeping the blocks black and background white would be the most optimal.
+- Size matters. You won't know the distance the devices will be from one another when the scan is happening, so you should design assuming the longest scan distance.
+
+## Waiting and transaction processing
```

Comment:
We probably need an illustration here.

Reply:
Even though the screens on the page already show QR codes?